rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -669,31 +766,39 @@
       "minLength": 2,
       "form": {
         "placeholder": "Example: 355 pages",
-        "hide": true
+        "hide": true,
+        "templateOptions": {
+          "cssClass": "col-lg-5"
+        }
       }
     },
     "duration": {
-      "title": "Duration",
+      "title": "Durations",
       "description": "A playing time, performance time, running time, etc., of the content of an expression.",
       "type": "array",
       "minItems": 1,
       "items": {
+        "title": "Duration",
         "type": "string",
         "minLength": 2,
         "form": {
           "placeholder": "Example: 1h50"
         }
       },
       "form": {
-        "hide": true
+        "hide": true,
+        "templateOptions": {
+          "cssClass": "rero-ils-editor-title"
+        }
       }
     },
     "colorContent": {
-      "title": "Colour content",
+      "title": "Color contents",
       "description": "A presence of colour, tone, etc., in the content of an expression. Record a colour content if considered important for identification or selection.",
```

Comment:
you can also fix "colour" here, can you check in the entire file?

Reply:
Done